.github/workflows/validate.yml

```diff
@@ -437,7 +382,9 @@ jobs:
     name: Validate post job
     runs-on: ubuntu-latest
     # IMPORTANT! Any job added to the workflow should be added here too
-    needs: [validate, validate-old-ghcs, build-alpine, dogfooding]
+    # (This one is true because of the abstraction described above, and the
+    # corresponding branch protection rules. ++bsa)
```

Comment:
Not sure what's "++bsa" about (here and above)

Reply:
Old habit, sorry… signing comments